pymeshfix/_version.py

```diff
@@ -7,5 +7,5 @@
 
 """
 
-version_info = 0, 17, "dev0"
+version_info = 0, 17, 0
```

Comment:
```suggestion
version_info = 0, 18, dev0
```

Reply:
Standard practice for pyvista repos is to keep main branch with the next minor ahead `dev0` (like numpy).